bin/scripts/org_migrator/org_migrator.py

```diff
@@ -207,13 +205,13 @@ def fetch_product_uuids(db, org_id):
     # logic layer, so it's not critical we only select distinct UUIDs with these queries. Further,
     # our second query must be partitioned and repeatedly executed, eliminating the value of any
     # DB-level deduplication feature anyway.
-    product_uuid_query = 'SELECT op.product_uuid FROM cp2_owner_products op WHERE op.owner_id = %s ' + \
+    product_uuid_query = 'SELECT op.uuid FROM cp_products op WHERE op.namespace = %s ' + \
```

Comment:
This won't catch global products unless they're used by a pool, which is supposed to be a catch-all for improperly mapped entities.